mtest/README.md

```diff
@@ -25,7 +25,9 @@ To see how the CI environment is doing it check `.travis.yml` and `build/run_tes
 
 **Note: You need to have `diff` in your path. For Windows, get a copy of [diffutils for Windows](http://gnuwin32.sourceforge.net/packages/diffutils.htm "diffutils for Windows").**
 
-**Note-2: For Windows MSVC, you also need MuseScore installation bin folder (for example C:\MuseScore\msvc.install_x64\bin) in your path.**
+**Note-2: For Windows MSVC and Windows MinGW you also need MuseScore installation bin folder (for example C:\MuseScore\msvc.install_x64\bin, C:\MuseScore\win32install\bin, C:\MuseScore\win64install\bin) in your path.**
```

Comment:
`Windows MinGW` is a tautology, the W in MinWG stands for Windows ;-)

Reply:
Addressed in b0365cb. Thank you! 👍